project/CSSGen.scala

```diff
@@ -3,8 +3,16 @@ import scala.sys.process._
 
 object CSSGen {
 
-  def genCssProp(name: String): String =
-    s"""  def `$name`(value: String): Style = Style("$name", value)"""
+  def genCssProp(name: String): String = {
+    val prop = s"""  def `$name`(value: String): Style = Style("$name", value)"""
+    if (!prop.contains("-") || prop.contains('@')) { prop }
+    else {
+      val namePC: String = name.split("-").map(_.capitalize).mkString
```

Comment:
Here you've got yourself into trouble early by capitalising everything, and you try and fix it later by doing a head + tail on the string.

If might work! But there's a cleaner way, which incidentally, also appears to work for properties beginning with `@`:

```scala
val name = "@my-fancy-prop"
name.split("-").toList match {
  case Nil =>
  	throw new Exception(s"CSS property '$name' was unexpectedly empty")

  case head :: tail =>
  	(head :: tail.map(_.capitalize)).mkString
}
```

Reply:
Yeah, that is a lot cleaner :) @ will have to be backticked anyway right? Because you can't declare a function in scala starting with @.

Reply:
Ah! Yes sorry that is a excellent point. :+1:

Reply:
Might be something to mention in the README/docs? I'm not sure if it's a problem specific enough to Tyrian to mention though, but could have a disclaimer like "Most properties have a camel case alias, except for properties that begin with @".